### PR TITLE
Another del->qdel pass

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Changes the vast majority of the remaining del's into qdel's.

There's still a few lurking in the code--mostly where del() is needed and it's not really worth defining a HARD_DEL_NOW (for instance, the map loader, master controller, process scheduler, or....things I'm not sure about like SQL2).

In any event, this leaves only three distinct areas specifically untouched from converting del->qdel:

- Holograms (runtimes out the butt)
- Projectiles/guns (infinite rounds)
- Stacks (weird behavior)

Most things qdel'd here won't have much if any of a performance boost as a lot are datums that are just going to be hard deleted anyway.

Fixes https://github.com/ParadiseSS13/Paradise/issues/2329

Also removes the completely unused sd_alerts library.

### CODE REVIEW

File: code/datums/modules.dm [hunk 1]
```diff
@@ -24,7 +24,7 @@ var/list/modules = list(			// global associative list
 	var/mneed = mods.inmodlist(type)		// find if this type has modules defined
 
 	if(!mneed)		// not found in module list?
-		del(src)	// delete self, thus ending proc
+		qdel(src)	// delete self, thus ending proc
```
Comment (tigercat2000) [inline code/datums/modules.dm]: given that the comment is literally `thus ending proc` this needs a return here

Comment (Fox-McCloud) [inline code/datums/modules.dm]: addressed, though Git doesn't think so for some reason.

File: code/modules/projectiles/projectile/beams.dm [hunk 2]
```diff
@@ -275,401 +275,4 @@
...
-		sleep(50)
-		src.invisibility = 101
-		return*/
-	on_hit(atom/target, blocked = 0)
-		if(istype(target, /mob/living))
-			var/mob/living/M = target
-			M.playsound_local(src, "explosion", 50, 1)
-		..()
-
-/obj/item/projectile/beam
```
Comment (tigercat2000) [inline code/modules/projectiles/projectile/beams.dm]: um

Comment (Fox-McCloud) [inline code/modules/projectiles/projectile/beams.dm]: all commented out; this is Bay's shitbeams.

**Comment by Fox-McCloud** [comment]
corrects a few del's too.

File: code/game/response_team.dm [hunk 3]
```diff
@@ -112,7 +112,7 @@ var/send_emergency_team
 				return
 			var/client/C = src.client
 			var/mob/living/carbon/human/new_commando = C.create_response_team(L.loc)
-			del(L) // Do not qdel landmarks!
```
Comment (TheDZD) [inline code/game/response_team.dm]: Same line that @tigercat2000 commented on, but Github handles comments on commits badly. Did the comment here perhaps have a point to it?

Comment (Fox-McCloud) [inline code/game/response_team.dm]: yes, landmarks don't qdel well, at all, but there's really no point in directly stating they don't qdel well, when it's implied with the landmark's Destroy(), which now returns a hint of HARDDEL_NOW.

**Pull request merged**